keyboards/dz60/info.json

```
@@ -1,2106 +1,1981 @@
 {
+    "manufacturer": "DZTECH",
     "keyboard_name": "DZ60",
-    "manufacturer": "KBDFans",
-    "url": "",
-    "maintainer": "qmk",
-    "usb": {
-        "vid": "0x445A",
-        "pid": "0x2260",
-        "device_version": "0.0.1"
-    },
-    "matrix_pins": {
-        "cols": ["F0", "F1", "E6", "C7", "C6", "B7", "D4", "B1", "B0", "B5", "B4", "D7", "D6", "B3", "F4"],
-        "rows": ["D0", "D1", "D2", "D3", "D5"]
-    },
-    "diode_direction": "COL2ROW",
-    "backlight": {
-        "pin": "B6",
-        "levels": 5
-    },
-    "indicators": {
-        "caps_lock": "B2",
-        "on_state": 0
-    },
-    "processor": "atmega32u4",
+    "maintainer": "DZTECH",
     "bootloader": "atmel-dfu",
+    "processor": "atmega32u4",
+    "debounce": 5,
+    "diode_direction": "COL2ROW",
     "community_layouts": ["60_ansi", "60_ansi_arrow_split_bs_7u_spc", "60_ansi_arrow", "60_ansi_split_bs_rshift", "60_hhkb", "60_iso", "60_abnt2", "60_tsangan_hhkb"],
```

add 64_* community layouts
```suggestion
    "community_layouts": ["60_ansi", "60_ansi_arrow_split_bs_7u_spc", "60_ansi_arrow", "60_ansi_split_bs_rshift", "60_hhkb", "60_iso", "60_abnt2", "60_tsangan_hhkb", "64_ansi", "64_iso"],
```